style/Alerts.css

```diff
@@ -0,0 +1,30 @@
+.alert-primary {
+    background-color: #7FD2E6;
+    color: #29438b;
+}
+.alert-secondary {
+    background-color: #eaeaec;
+    color: #696b79;
+}
+.alert-success {
+    background-color: #84f1c9;
+    color: #1b7052;
+}
+.alert-danger {
+    color: #dc3545;
+    background-color: #fcc5c5;
+}
+.alert-warning {
+    color: #ff9204;
+    background-color: #ffe7c5;
+}
+
+.alert-items{
+    padding:10px 10px 10px 10px;
+    margin:20px 0px 20px 0px;
+    border-radius: 10px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
+}
+.alert-icon{
+    padding:0px 10px 0px 10px
+}
```

Comment:
use root variables for colors

Reply:
I have used

Reply:
sorry the wrong line I selected I thing😅